modules/040-node-manager/images/early-oom/src/psi-monitor.go

```diff
@@ -0,0 +1,120 @@
+/*
+Copyright 2021 Flant JSC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"log"
+	"os"
+	"os/signal"
+	"regexp"
+	"strconv"
+	"syscall"
+	"time"
+)
+
+var fullAvg10Regex = regexp.MustCompile(`^full avg10=(\d{1,}\.\d{2}).*$`)
+
+const (
+	pollInterval     = 5 * time.Second
+	recoveryInterval = 15 * time.Second
+	memoryThreshold  = 5.00
```

Comment:
1. Is this a percentage? 
2. Do we need to configure it differently for nodes with different memory capacity?